enclave/core/sgx/asmdefs.h

```diff
@@ -17,7 +17,20 @@
 #define STATIC_STACK_SIZE 8 * 100
 #define OE_WORD_SIZE 8
 
-#define CODE_ERET 0x200000000
+#define CODE_EXCEPTION_CONTINUE_EXECUTION 0xFFFFFFFF
+
+/* match oe_td_state_t defined in internal/sgx/td.h */
```

Comment:
Can you augment the comment to explain why the enum values of oe_td_state_t itself cannot be used here and defines are instead needed?